### PR TITLE
Add ML FE to documentation

### PR DESCRIPTION
**Pull request by JTeijema**
This PR will add the multilingual feature extractor to the documentation.

### CODE REVIEW

**Comment by JTeijema** [comment]
The updates docs look like this: 
image

**Review by Rensvandeschoot — APPROVED** [review]

File: docs/source/extensions/overview_extensions.rst [hunk 1]
```diff
@@ -21,11 +21,20 @@ of the core project:
...
 
-* Feature Extraction
+* Feature Extraction Vocab Extractor
     - `ASReview-vocab-extractor <https://github.com/asreview/asreview-extension-vocab-extractor>`__: 
-      This extension adds two feature extractors that extract vocabulary and 
-      vector matrices during simulation phases. Might one day be integrated to the 
-      core.
+      This extension adds two feature extractors that extract vocabulary and
+      vector matrices during simulation phases for research purposes. Might one
+      day be integrated to the core.
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:28]:
```suggestion
      This extension adds two feature extractors that extract vocabulary and
      vector matrices during simulation phases for research purposes.
```

Comment (J535D165) [reply docs/source/extensions/overview_extensions.rst:28]: Might be better to file an issue about this and discuss it over there.

File: docs/source/extensions/overview_extensions.rst [hunk 2]
```diff
@@ -21,11 +21,20 @@ of the core project:
...
+* Multilingual Feature Extraction
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:29]:
```suggestion
* Feature Extraction Multilingual
```

Comment (J535D165) [reply docs/source/extensions/overview_extensions.rst:29]: Consistent with other one.

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks

**Review by Rensvandeschoot — APPROVED** [review]

**Review by J535D165 — APPROVED** [review]

**Pull request merged**